datasets/nrcan-landcover/nrcan-landcover-example.ipynb

```diff
@@ -0,0 +1,1352 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "merged.plot()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b105bbe2",
+   "metadata": {},
+   "source": [
+    "It looks good, but it doesn't look like a land cover map. The source GeoTIFFs contain a colormap and the STAC metadata contains the class names. We'll open one of the source files just to read this metadata and construct the right colors and names for our plot."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 108,
+   "id": "03fb3c5a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Replace nodata values with np.nan\n",
```

Comment:
We really need to get this colormap information in STAC :) This is good for now though.

Reply:
Yeah, it feels a bit awkward at the moment! Especially so for `nrcan-landcover` which doesn't have a neat sequential mapping between class numbers and names.